ToDoApp/ListViewController/ListViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  ToDoTable
+//
+//  Created by deniss.lobacs on 15/02/2022.
+//
+
+import UIKit
+
+class ListViewController: UIViewController {
+    
+    var tableView = UITableView()
+    var listViewModel: ListViewModel?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configTable()
+    }
+        
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        tableView.reloadData()
```

Comment:
why do you need it?

Reply:
for reloading tableview after appending inside new element

Reply:
Are you sure, that this method will be called after appending new element? let's create a public function updateTableView() and call it when it necessary